### PR TITLE
netbird: Add version 0.29.4

### PR DESCRIPTION
**Pull request by Cikaros**
add bin Netbird 0.29.3
fix version and hash

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Cikaros** [comment]
Has been tested in Windows 10 and can be used normally.
image
image
image

File: bucket/netbird.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.29.3",
+    "description": "Connect your devices into a secure WireGuard®-based overlay network with SSO, MFA and granular access controls.",
+    "homepage": "https://github.com/netbirdio",
```
Comment (aliesbelik) [inline bucket/netbird.json:4]:
```suggestion
    "homepage": "https://netbird.io/",
```

**Review by aliesbelik — COMMENTED** [review]
Please update to the latest version, otherwise looks good.

**Review by Cikaros — COMMENTED** [review]
Update version to 0.29.4

File: bucket/netbird.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "bin": "netbird.exe",
+    "shortcuts": [
+        [
+            "netbird.exe",
+            "netbird"
+        ]
+    ],
+    "checkver": {
+        "github": "https://netbird.io/"
```
Comment (aliesbelik) [inline bucket/netbird.json:20]: Any reason to switch from github link?

**Pull request merged**